charmcraft/extensions/app.py

```diff
@@ -31,7 +31,7 @@ class _AppBase(Extension):
         {"lib": "traefik_k8s.ingress", "version": "2"},
         {"lib": "observability_libs.juju_topology", "version": "0"},
         {"lib": "grafana_k8s.grafana_dashboard", "version": "0"},
-        {"lib": "loki_k8s.loki_push_api", "version": "0"},
+        {"lib": "loki_k8s.loki_push_api", "version": "1"},
```

Comment:
Please address the test issues this raised :-)

Reply:
@lengau it looks like the Python 3.12 tests weren't even run?

Reply:
I have fixed all the issues with tests, thanks!

Reply:
@jdkandersson the setup is a bit weird right now, as we use tox and skip environments inside it, but across all the Linux versions it runs all the Python tests. We're moving towards a base testing system that'll be easier to follow across all of our repos.